Disjoint_Sets/Disjoint_Sets.java

```diff
@@ -0,0 +1,66 @@
+public class Disjoint_Sets {
+    //declare global arrays to store the parent of each element and size of each set
+    static int[] parent;
+    static int[] size;
+    
+    //function that initially defines the parent of all element as itself.
+    static void makeSet(int n) {
+        for (int i=0;i<n+1;i++) {
```

Comment:
Add a space after ``;`` and always follow coding conventions (https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html).
Add space around operators according to coding conventions throughout this program.